api/stripe.go

```diff
@@ -182,17 +182,16 @@ func readStripeEvent(w http.ResponseWriter, req *http.Request) (*stripe.Event, i
 // adjusts the user's record accordingly.
 func (api *API) processStripeSub(ctx context.Context, s *stripe.Subscription) error {
 	api.staticLogger.Traceln("Processing subscription:", s.ID)
+	u, err := api.staticDB.UserByStripeID(ctx, s.Customer.ID)
+	if err != nil {
+		return errors.AddContext(err, "failed to fetch user from DB based on subscription info")
```

Comment:
`nit` could improve the error here and say "no user found from stripe customer with ID '%v'", would be annoying to find that error without a mention of the stripe customer :)